.github/workflows/rm_gh_cache_after_pr_closed.yml

```diff
@@ -0,0 +1,45 @@
+name: Cleanup caches after PR closed
+on:
+  pull_request:
+    # branches:
+    #   - dev
+    types:
+      - closed
+  workflow_dispatch:
```

Comment:
What's the branch if it's triggered by `workflow_dispatch` (I assume it's null), then run logic will be broken.

Reply:
Yes. Should be removed. Not necessary.